Samples/HelloWorld/HelloWorld.SeparateProcess.sln

```diff
@@ -0,0 +1,40 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio 14
+VisualStudioVersion = 14.0.25420.1
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "HelloWorldInterfaces", "HelloWorldInterfaces\HelloWorldInterfaces.csproj", "{3BD01348-ACC3-4378-A49E-4D62FB0BD3D0}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "HelloWorldGrains", "HelloWorldGrains\HelloWorldGrains.csproj", "{943794E3-F54E-4678-B225-B45018344AEC}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "OrleansClient", "OrleansClient\OrleansClient.csproj", "{05CB5084-F694-4CD1-A4A3-C08ABE8B4476}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "OrleansHost", "OrleansHost\OrleansHost.csproj", "{76521F11-81CD-46D0-970F-2F972F67F626}"
```

Comment:
should we call the project something different than OrleansHost? Otherwise this will generate an OrleansHost.exe file which can be confused with the pre-built OrleansHost.exe that comes in the OrleansHost nuget package.
Maybe SiloHost?